app/helpers/automated_tests_client_helper.rb

```diff
@@ -196,7 +196,7 @@ def self.get_test_server_user
   # Verify the user has the permission to run the tests - admins
   # always have the permission, while student has to
   # belong to the group, and have at least one token.
-  def self.check_user_permission(user, grouping)
+  def self.check_user_permission(user, grouping, assignment)

```

Comment:
I can change it.
I just had it already available in the outer scope and don't know Rails' caching enough to be sure it wasn't triggering a query.